### ISSUE TITLE
prosper playbook not accessible 

### ISSUE THREAD

**Issue by Bonifacio2** [johnellison/john-ellison.com#2]
In http://john-ellison.com/prosper-community.html, when you mention Prosper's playbook you included it as an image. Whoever access this page using a screen reader will be unable to access the playbook's content. I know this is not the only image there and when it comes to the other diagrams there's no easy way to describe their contents, but there's an easy fix for the playbook accessibility: just replace the image with plain text.
:)

**Comment by johnellison** [comment]
Thanks for this also @Bonifacio2, I've fixed this in https://github.com/johnellison/john-ellison.com/commit/55ba30fb2220e406c1691c12fe5db146300b386c.
